lib/entities/comment.ts

```diff
@@ -34,6 +41,38 @@ export type UpdateCommentProps = Omit<CommentProps, 'sys'> & {
   sys: Pick<CommentSysProps, 'version'>
 }
 
+export enum CommentNode {
+  Document = 'document',
+  Paragraph = 'paragraph',
+  Mention = 'mention',
+}
+
+export interface MentionNodeContent extends Node {
+  nodeType: 'text'
+  value: string
+  marks: Mark[]
+}
+
+export interface Mention {
+  nodeType: CommentNode.Mention
+  data: { target: Link<'User'> }
+  content: MentionNodeContent[]
+}
```

Comment:
[change request]
Is there reason not to use the original type `Text` from the rich-text package? You can find it [here](https://github.com/contentful/rich-text/blob/47f96df347c73ac301cbcb39dea23dc9236d46c6/packages/rich-text-types/src/types.ts#L34).